docs/content/templates/single-page-form/index.mdx

```diff
@@ -43,8 +43,12 @@ Instead, allow users to interact with buttons and receive feedback.
 
 ### Error messages
 
-A form should validate when a user attempts to submit. If there are errors, they should be listed in an error summary (or [PageAlert](/components/page-alert)) at the top of the page. The summary should link to each error and focus on the form field when clicked.
+A form should validate when a user attempts to submit. If there are errors, they should be listed in an error summary (or [PageAlert](/components/page-alert)), placed at the top of the page and immediately focused. The summary should link to each error and focus on the form field when clicked.
 
 If only one error is present, the error summary can be omitted and the invalid field can be focused.
 
 An invalid field is highlighted with a red border and a red error message. They should never be covered by autocomplete menus and on-screen keyboards.
+
+### Success messaging
+
+When a form is successfully completed, return the user to the entry point of the form and display a [PageAlert](/components/page-alert) with the green 'success' tone and a clear message which communicates the successful submission. When possible provide, persistent methods for a user to return and see the status of the from submission - ie a reference number for the artifact.
```

Comment:
```suggestion
When a form is successfully completed, return the user to the entry point of the form and display a [PageAlert](/components/page-alert) with the 'success' tone and a clear message which communicates the successful submission. When possible, provide persistent methods for a user to return and see the status of the submitted artefact, for example a reference number.
```

Reply:
- I don't think we need to list the word 'green'
- Fixed a couple of small typos
- Rearranged the last sentence